src/main/java/edu/byu/cs/autograder/score/Scorer.java

```diff
@@ -202,9 +201,11 @@ private boolean passed(Rubric rubric) {
         boolean passed = true;
 
         boolean isPassoffRequired = PhaseUtils.isPassoffRequired(gradingContext.phase());
-        if (isPassoffRequired && rubric.passoffTests() != null && rubric.passoffTests().results() != null)
-            if (rubric.passoffTests().results().score() < rubric.passoffTests().results().possiblePoints())
-                passed = false;
+        var passoffTestItem = rubric.items().get(Rubric.RubricType.PASSOFF_TESTS);
+        if (isPassoffRequired && passoffTestItem != null && passoffTestItem.results() != null &&
```

Comment:
I know that the single-exit point execution is sometimes desirable, but I think this code would be a lot more readable with guarded exits.

I.E.
```java
if (!isPassoffRequired) return true;
...
if (passoffTestItem == null || passoffTestItem.results() == null) return true;
...
return passoffTestItem.results().score() >= passoffTestItem.results().possiblePoints();
```

Reply:
Seems reasonable. How does that look?